Radzen.Blazor/DropDownBase.cs

```diff
@@ -971,6 +972,19 @@ protected void OnChange(ChangeEventArgs args)
             internalValue = args.Value;
         }
 
+        /// <summary>
+        /// Method that is triggered when the popup opens.
+        /// </summary>
+        [Parameter]
+        public Action OnOpenPopup { get; set; }
```

Comment:
Please use EventCallback isntead Action and remove On prefix. Such prefixes are usually used for the event handler not for the event itself.

Reply:
Hi @enchev, the prefix On was removed from the parameters, instead I added the noun Callback since I can't simply leave the property name as OpenPopup because it conflicts with the name of the internal method

Reply:
Ok, @enchev  I've changed the name of the internal method from OpenPopup to TogglePopup.